DragaliaAPI.Database/Repositories/UnitRepository.cs

```diff
@@ -103,6 +112,38 @@ IEnumerable<Charas> idList
             );
 
             await apiContext.PlayerCharaData.AddRangeAsync(dbEntries);
+
+            List<DbPlayerStoryState> newCharaStories = new List<DbPlayerStoryState>(
+                newCharas.Count()
+            );
+            for (int i = 0; i < newCharas.Count(); i++)
+            {
+                if (
+                    MasterAsset.CharaStories.TryGetValue(
+                        (int)newCharas.ElementAt(i),
+                        out StoryData? story
+                    )
+                )
+                {
+                    newCharaStories.Add(
+                        new DbPlayerStoryState()
+                        {
+                            DeviceAccountId = deviceAccountId,
+                            StoryType = StoryTypes.Chara,
+                            StoryId = story.storyIds[0],
+                            State = 0
+                        }
+                    );
+                }
+                else
+                {
+                    logger.LogWarning(
```

Comment:
Can you make this information as it doesn't necessarily indicate an error or unexpected behaviour

Or maybe log `adding chara X`, then `adding stories Y and Z`, then just not logging the latter in this case

Reply:
Addressed in 1b57798